src/aap_eda/services/activation/activation_manager.py

```diff
@@ -393,7 +393,7 @@ def _unresponsive_policy(self, check_type: str):
             )
             user_msg = (
                 "Activation is unresponsive. "
-                f"{check_type} check for ansible rulebook timed out. "
+                f"{check_type} check for ansible-rulebook timeout. "
```

Comment:
Hmm strictly speaking "timed out" is correct, as it's the past participle of "time out"

Reply:
Good catch, I misread it